src/AccessibilityInsights.RulesTest/PropertyConditions/ElementGroupsTests.cs

```diff
@@ -61,5 +61,110 @@ public void AllowSameNameAndControlType_True_ExceedsMaxLength()
 
             Assert.IsTrue(ElementGroups.AllowSameNameAndControlType.Matches(e));
         }
+
+        [TestMethod]
+        public void WPFScrollbarPageUpButton_True()
+        {
+            using (var e = new MockA11yElement())
+            using (var parent = new MockA11yElement())
+            {
+                parent.ControlTypeId = ControlType.ScrollBar;
+                e.IsOffScreen = false;
+                e.ControlTypeId = ControlType.Button;
+                e.Framework = "WPF";
+                e.AutomationId = "PageUp";
+                parent.Children.Add(e);
+                e.Parent = parent;
+
+                Assert.IsTrue(ElementGroups.WPFScrollBarPageUpOrPageDownButtons.Matches(e));
+            } // using
+        }
+
+        [TestMethod]
+        public void WPFScrollbarPageDownButton_True()
+        {
+            using (var e = new MockA11yElement())
+            using (var parent = new MockA11yElement())
+            {
+                parent.ControlTypeId = ControlType.ScrollBar;
+                e.ControlTypeId = ControlType.Button;
+                e.Framework = "WPF";
+                e.AutomationId = "PageDown";
+                parent.Children.Add(e);
+                e.Parent = parent;
+
+                Assert.IsTrue(ElementGroups.WPFScrollBarPageUpOrPageDownButtons.Matches(e));
+            } // using
+        }
+
+        [TestMethod]
+        public void WPFScrollbarPageUpOrPageDownButtons_NotWPF_False()
```

Comment:
>PageUpOrPageDown [](start = 32, length = 16)

Name suggests this tests both PageUp and PageDown, when it tests only PageUp

Reply:
No longer applicable.